app/ydoc-shared/src/ast/tree.ts

```diff
@@ -199,7 +199,7 @@ export abstract class Ast {
   }
 
   static parseBlock(source: string, inModule?: MutableModule) {
-    return parseBlock(source, inModule)
+    return parseBlock(source, { module: inModule })
```

Comment:
Parameter "inModule" strongly suggests that this is a "body block" (it's "in the module", not "the entire module") - but actually it's the other way round.

Reply:
It is always a body block here. If the `inModule` parameter is provided AST objects will be associated with the given `MutableModule`, otherwise a new `MutableModule` will be used; this has no effect on how the input is parsed.

Reply:
Ah, yes, I misread this code